selenium/launch_cards.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+CARDS_PID=$(pgrep "start_cards.sh")
+
+function check_cards_running() {
+  docker ps | grep compose-cluster-cardsinitial-1 > /dev/null
+}
+
+function handle_cards_java_fail() {
+  echo -e "Cards has failed to start, cannot run selenium tests"
+  exit -1
+}
+
+echo "Cleaning up old CARDS docker images"
+docker-compose down && docker-compose rm && docker volume prune -f && ./cleanup.sh
+echo "Generating docker configuration"
+python3 generate_compose_yaml.py --oak_filesystem --dev_docker_image
+echo "Launching CARDS docker image"
+docker-compose build > /dev/null && docker-compose up -d
```

Comment:
```suggestion
docker-compose build > /dev/null && docker-compose up -d || exit -1
```

Suppose that some other service, perhaps another CARDS instance, is already running on port `127.0.0.1:8080`. In such a case, `docker-compose up -d` would fail and therefore so should this script otherwise it could incorrectly conclude that CARDS had started because _another_ CARDS instance was already available at `127.0.0.1:8080` - and we want to be sure that we're testing _this_ build.